CHANGELOG.md

```diff
@@ -11,10 +11,15 @@ This document follows the guidelines of [Keep A Changelog](http://keepachangelog
 ### Added
 * ESLint (...finally!) (#305) (@Zearin)
 
-
 ### Removed
-* Dependency `absolute` (save ≈21k!) (c05f9e2) (@Zearin)
+* **Remove Dependencies:**
+  - `absolute` (save ≈21k!) (c05f9e2) (@Zearin)
+  - `is` (save ≈74k!) (7eaac9e2, 54dba0c1) (@Zearin)
 
+### Changed
+* When a `Metalsmith` setter receives an invalid argument, it now throws 
+  a `TypeError` instead of an `AssertionError` (but error messages 
+  are unchanged) (@Zearin)
```

Comment:
Hmm... now that I think about it, this _is_ technically an API change. I can't imagine many people are intentionally passing invalid argument types to Metalsmith and checking whether the errors are `instanceof AssertionError` though, so _maybe_ this is okay for a minor release? I don't know, it's a really minor thing but maybe we should change this back until the next major release?

Reply:
Yeah, I had the same thought.  I didn’t think it would matter, so I figured I’d let you decide in review.  

Should I revert?

Reply:
Hmm... yeah might as well revert that change for now. I doubt it would break anything for anyone, but better safe than sorry.

Reply:
Ah, okay.  Alas!  The burden of backward-compatibility.  👅